fhir-install/Dockerfile

```diff
@@ -29,6 +29,7 @@ USER 1001
 
 FROM openliberty/open-liberty:21.0.0.3-kernel-slim-java8-openj9-ubi
 
+ARG VERBOSE=true
```

Comment:
maybe set build arg to false by default?

Reply:
I've found the VERBOSE output to be useful to see whats going on, and I don't see much downside to including it